src/partials/sidebar/index.js

```diff
@@ -143,11 +143,14 @@ const Sidebar = ({ className, content, location, onCloseClick }) => {
         filteredResults.filter(edge => {
           return edge.data && edge.data.type === sectionCategory.type;
         });
-      return (filteredByCategory[
-        [sectionCategory.category]
-      ] = filteredEdges.filter(edge =>
-        sectionCategory.category.includes(edge.data.category)
-      ));
+
+      return filteredEdges
```

Comment:
Fixes the `Failed exporting HTML for URL gallery/area-hover-styles (/Users/${user-repo-path}/victory-docs/src/pages/gallery-item-template): Cannot read property 'filter' of undefined` issue